pkg/patterns/addon/pkg/loaders/types.go

```diff
@@ -132,21 +132,25 @@ func (r *FSRepository) LoadManifest(ctx context.Context, packageName string, id
 }
 
 type Channel struct {
-	Manifests []Version `json:"manifests,omitempty"`
+	Packages []Version `json:"packages,omitempty"`
 }
 
 type Version struct {
-	Version string
+	Package string `json:"name"`
+	Version string `json:"version"`
 }
 
-func (c *Channel) Latest() (*Version, error) {
+func (c *Channel) Latest(packageName string) (*Version, error) {
 	var latest *Version
-	for i := range c.Manifests {
-		v := &c.Manifests[i]
+	for i := range c.Packages {
+		v := &c.Packages[i]
+		if v.Package != "" && v.Package != packageName {
+			continue
+		}
 		if latest == nil {
 			latest = v
-		} else {
-			return nil, fmt.Errorf("version selection not implemented")
+		} else if latest.Compare(v) < 0 {
```

Comment:
In my lab environment, Error occurs following.
Missing definition of function?
```
latest.Compare undefined (type *Version has no field or method Compare)
```

Reply:
Thanks - I had mixed up my branches!

Reply:
I've checked it. :+1: